yaml_files/simphony_metadata.yml

```diff
@@ -438,9 +513,12 @@ CUDS_KEYS:
       scope: CUBA.USER
 
   BOX:
-    definition: A simple hexahedron box object
+    definition: A simple hexahedron (with six faces) simulation box defined by the three vectors and three directions. The condition should be specified for each direction (two faces at a time).   
```

Comment:
we should aim to be consistent and if 3 vectors are represented by a matrix, then it should be row wise always! 
changed the comment in the file.